src/contentscript/overlay/root/components/OverlayTab/index.tsx

```diff
@@ -56,13 +57,31 @@ export const OverlayTab = (p: OverlayTabProps): ReactElement => {
   const visibleMenus = p.menus.filter((x) => x.hidden !== true)
   const nodeVisibleMenu = useRef<HTMLDivElement>()
   const [menuVisible, setMenuVisible] = useState(false)
+  const [event, setEvent] = useState<Notify[]>([])
 
   useEffect(() => {
     !document
       .querySelector('#dapplets-overlay-manager')
       .classList.contains('dapplets-overlay-collapsed') && setMenuVisible(false)
+    return () => {}
   }, [menuVisible])
 
+  useEffect(() => {
+    const handleUpdateNotifications = async () => {
+      const notifications = await getNotifications()
+      setEvent(notifications && notifications.filter((x) => x.status === 1))
+    }
+
+    handleUpdateNotifications()
+    EventBus.on('notifications_updated', handleUpdateNotifications)
+    EventBus.on('show_notification', handleUpdateNotifications)
```

Comment:
Is notifications_updated not enough?

Reply:
not enough
Animate is used when notification show